processors/filtering/base_filter.py

```diff
@@ -79,6 +78,13 @@ def after_process(self):
         # Request standalone
         self.create_standalone()
 
+    @classmethod
```

Comment:
Oh, this should probably have been in the `master` branch. Quicker than the processor class's method of figuring it out.